pkg/kcp/provider/azure/redisinstance/loadPrivateEndPoint.go

```diff
@@ -44,6 +44,7 @@ func loadPrivateEndPoint(ctx context.Context, st composed.State) (error, context
 		}
 		return composed.StopWithRequeueDelay(util.Timing.T60000ms()), nil
 	}
+	logger.Info("Azure Private EndPoint instance loaded", "provisioning state", privateEndPointInstance.Properties.ProvisioningState)
```

Comment:
I don't know how to write GCP log query to filter by property with space in its name. So far we used camelCase
```suggestion
	logger.Info("Azure Private EndPoint instance loaded", "provisioningState", privateEndPointInstance.Properties.ProvisioningState)
```